app/mailers/jobapps_mailer.rb

```diff
@@ -42,6 +42,16 @@ def interview_reschedule(interview)
          reply_to: reply_to
   end
 
+  def send_note_for_later(application_record)
```

Comment:
How about a mailer method that emails the Staff user tied to this Position when a application comes out of `saved_for_later`?  This might be a question for Don as well.